lua/spectre/init.lua

```diff
@@ -545,10 +545,12 @@ M.change_options = function(key)
         state.options[key] = false
     end
     state.options[key] = not state.options[key]
-    if #state.query.search_query > 0 then
-        ui.render_search_ui()
-        M.search()
-    end
+    pcall(function()
+        if #state.query.search_query > 0 then
```

Comment:
just check search_query is not null and remove pcall .pcall will hide another error.